js/main.js

```diff
@@ -0,0 +1,47 @@
+var $codeJournal = document.querySelector('#code-journal');
+var $placeHolderImage = document.querySelector('.placeholder-image');
+var $photoUrlInput = document.querySelector("input[name='photoUrl']");
+var dataModel = data;
```

Comment:
I'm not sure what the purpose is of this, I would just stick to using the `data` variable.